.eslintrc.js

```diff
@@ -106,6 +109,16 @@ module.exports = {
 		} ],
 	},
 	overrides: [
+		{
+			files: [ 'packages/**/*.js' ],
+			rules: {
+				'import/no-extraneous-dependencies': 'error',
+			},
+			excludedFiles: [
+				'**/*.@(android|ios|native).js',
```

Comment:
@hypest and @koke - I disabled native files for now to fix all the issues which web version has, but we definitely should revisit it really soon.

Reply:
there is a follow-up: #16978